### PR TITLE
Fuzzy date pipe

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Addresses #100

### CODE REVIEW

File: src/app/visual/date-pipe/date-pipe-visual.component.html [hunk 1]
```diff
@@ -1,52 +1,119 @@
 <div
   class="app-screenshot"
+  style="width:300px"
 >
+
+  <h2>
+    Date pipe
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/date-pipe/date-pipe-visual.component.html]: Note: The changes to the date pipe visual were intended to be an upgrade to make the intent and manual testing of the pipe easier. I made the fuzzy date pipe this way, and decided to give it a whirl on the regular date pipe. Let me know what you think!

**Note: There are no visual tests for both the date pipe and fuzzy date pipe. All coverage is done in unit.**

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-datetime?_sp.spa=pr-fuzzy-date-pipe](https://host.nxt.blackbaud.com/skyux-datetime?_sp.spa=pr-fuzzy-date-pipe)

File: src/app/index.html [hunk 2]
```diff
@@ -12,20 +12,29 @@ <h2>
   Visual tests
 </h2>
 
+<h3>Pickers</h3>
 <ul>
   <li>
-    <a skyAppLink="/visual/date-pipe">Date pipe</a>
+    <a href="#" skyAppLink="/visual/datepicker">Datepicker</a>
```
Comment (Blackbaud-SteveBrush) [inline src/app/index.html]: We can remove these `href="#"`.

File: src/app/public/modules/date-pipe/fuzzy-date.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+import {
+  SkyFuzzyDateService
+} from '../datepicker/fuzzy-date.service';
+
+/**
+ * Formats a fuzzy date value according to locale rules.
+ * You can construct a format string using symbols to specify the components
+ * of a date-time value, as described in the following table.
+ * These fields are based on the moment.js values found here: https://momentjs.com/docs/#/displaying/
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/date-pipe/fuzzy-date.pipe.ts]: `These fields are based on the [moment.js values](https://momentjs.com/docs/#/displaying/).`

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-fuzzy-date-pipe](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-fuzzy-date-pipe)

File: src/app/public/modules/date-pipe/fuzzy-date.pipe.ts (hunk 3 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/date-pipe/fuzzy-date.pipe.ts]: done.

File: src/app/index.html (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/index.html]: Done.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-fuzzy-date-pipe](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-fuzzy-date-pipe)

**Pull request merged**